### PR TITLE
Participant Attendance Improvement

### LINKED ISSUE #367: Registrant Attendance UX
image

registrant attendance is really hard to understand, it need to fixed so it become more usable/understandable.
fix option are 2, there is:
- Using tab to change between timeline, and give header action named Mark In day to attendance in it
- Using custom view so its more flexible on design etc.

### PR DESCRIPTION
**Pull request by SalimQS**
Preview:

Per-day attendance (Timeline)
image

Per-session attendance (Session)
image

### CODE REVIEW

**Comment by rahmanramsi** [comment]
CleanShot 2024-09-02 at 08 10 14
Default value attendance time jam saat ini

File: app/Providers/PanelProvider.php [hunk 1]
```diff
@@ -156,9 +156,9 @@ public function setupPanel(Panel $panel): Panel
     {
         return $panel
             ->maxContentWidth('full')
-            ->when(!app()->isProduction(), fn(Panel $panel) => $panel->renderHook(
-                PanelsRenderHook::FOOTER,
-                fn() => Blade::render('<x-livewire-handle-error />')))
+            // ->when(!app()->isProduction(), fn(Panel $panel) => $panel->renderHook(
+            //     PanelsRenderHook::FOOTER,
+            //     fn() => Blade::render('<x-livewire-handle-error />')))
```
Comment (rahmanramsi) [inline app/Providers/PanelProvider.php]: knp di disable?

**Review by rahmanramsi — CHANGES_REQUESTED** [review]

File: app/Panel/ScheduledConference/Resources/RegistrantResource/Pages/ParticipantAttendance (temporary).php
Comment (rahmanramsi) [inline app/Panel/ScheduledConference/Resources/RegistrantResource/Pages/ParticipantAttendance (temporary).php:1]: file yg tidak terpakai hapus aja

File: app/Providers/PanelProvider.php (hunk 1 above)
Comment (SalimQS) [reply app/Providers/PanelProvider.php]: maaf ka lupa dikembalikan ke awal

File: app/Panel/ScheduledConference/Resources/RegistrantResource/Pages/ParticipantAttendance (temporary).php
Comment (SalimQS) [reply app/Panel/ScheduledConference/Resources/RegistrantResource/Pages/ParticipantAttendance (temporary).php:1]: baik ka

**Review by rahmanramsi — APPROVED** [review]

**Pull request merged**